### PR TITLE
Consistent link delimiters

### PR DESCRIPTION
**Pull request by prees1**
This PR will resolve this issue - https://github.com/tidalmigrations/tidalmigrations.github.io/issues/82

Before merging this, need to confirm any external references to these older URLs are updated.

### CODE REVIEW

**Comment by dcolebatch** [comment]
Case closed?

**Comment by prees1** [comment]
Oh, itis not merged because we need to either support redirects from the existing links to the new ones, or update to using the new ones in other places.

**Comment by dcolebatch** [comment]
ohhh I see - well, I resolved your merge conflict there to use `-` for urls (`export-ip-addresses.html`)

**Comment by justinbarclay** [comment]
> Oh, it's not merged because we need to either support redirects from the existing links to the new ones, or update to using the new ones in other places.

Is this resolved or still an open question?

File: _data/sidebars/main_sidebar.yml [hunk 1]
```diff
@@ -51,16 +51,16 @@ entries:
       url: analyze-source-code.html
     - title: Accessing Source Code Premium Insights
       output: web
-      url: /source_code_insights.html
+      url: /source-code-insights.html
```
Comment (justinbarclay) [inline _data/sidebars/main_sidebar.yml:54]: Why do some of these URLs have a forward slash and others don't? Are they a nested route?

**Comment by prees1** [comment]
Still pending. Mainly need to preserve (either update or redirect) links from app.tidalmg.com or tidalmigrations.com

File: _data/sidebars/main_sidebar.yml (hunk 1 above)
Comment (prees1) [reply _data/sidebars/main_sidebar.yml:54]: no good reason, both are valid. we have been inconsistent when adding them.

**Comment by pshevtsov** [comment]
It is impossible to add nested links to the sidebar. When you add something like foo/bar/baz/apple/fox.html it renders as a link to foobarbazapplefox.html 🤷‍♂️

Petr Shevtsov

**Pull request merged**